l10n/README.md

```diff
@@ -0,0 +1,316 @@
+    # TRANSLATOR_NOTE: Option when scanning for a matching address; skips ten addresses ahead
+    SKIP_10 = ButtonOption("Skip 10")
+```
+
+Note that the comment MUST be on the preceding line of executable code for it to work:
+
+```python
+class SettingsConstants
+    # TRANSLATOR_NOTE: QR code density option: Low, Medium, High       <-- ✅ Correct way to add context
+    density_low = _mft("Low")
+
+    ALL_DENSITIES = [
+        (DENSITY__LOW, density_low),
+        # TRANSLATOR_NOTE: QR code density option: Low, Medium, High   <-- ❌ Note will NOT be picked up
+        (DENSITY__MEDIUM, "Medium"),
+        (DENSITY__HIGH, "High"),
+    ]
+```
+
+```python
+# TRANSLATOR_NOTE: Refers to the user's change output in a psbt
+some_var = _("change")
+```
+
+---
+
+## `_()` Wrapping syntax details
+* Use `.format()` to wrap strings with variable injections. Note that `.format()` is OUTSIDE the `_()` wrapping.
+    ```python
+    mystr = f"My dad's name is {dad.name} and my name is {self.name}."
+    mystr = _("My dad's name is {} and my name is {}").format(dad.name, self.name)
+    ```
+
+    The translators will only see: "My dad's name is {} and my name is {}" in Transifex. Often the English string literal is 
+    basically incomprehensible on its own so always provide an explanation for what is being injected:
+
+    ```python
+    # TRANSLATOR_NOTE: Address verification success message (e.g. "bc1qabc = seed 12345678's receive address #0.")
+    text = _("{} = {}'s {} address #{}.").format(...)
+    ```
+
+    If there are a lot of variables to inject, placeholder names can be used (TODO: how does Transifex display this?):
+    ```python
+    mystr = _("My dad's name is {dad_name} and my name is {my_name}").format(dad_name=dad.name, my_name=self.name)
+    ```
+* Use `ngettext` to dynamically handle singular vs plural forms based on an integer quantity:
+    ```python
+    n = 1
+    print(ngettext("apple", "apples", n))
+    >> apple
+    
+    n = 5
+    print(ngettext("apple", "apples", n))
+    >> apples
+    ```
+
+Transifex will ask translators to provide the singular and plural forms on a language-specific basis (e.g. Arabic as THREE plural forms!).
+
+---
+
+## Set up localization dependencies
+```bash
+pip install -r l10n/requirements-l10n.txt
+```
+
+Make sure that your local repo has fetched the `seedsigner-translations` submodule. It's configured to add it in src/seedsigner/resources.
+```bash
+# Need --remote in order to respect the target branch listed in .gitmodules
+git submodule update --remote
+```
+
+
+### Pre-configured `babel` commands
+The `setup.cfg` file in the project root specifies params for the various `babel` commands discussed below.
+
+You should have already added the local code as an editable project in pip:
+```bash
+# From the repo root
+pip install -e .
+```
+
+
+### Rescanning for text that needs translations
+Re-generate the `messages.pot` file:
+
+```bash
+python setup.py extract_messaages
```

Comment:
@kdmukai I'm pretty sure that `extract_messaages` is a typo and it should be `extract_messages`

Reply:
Fixed!